### PR TITLE
Fixed _handle_range

### PR DESCRIPTION
**Pull request by pizzapanther**
### What does this PR do?

Fixes `_handle_range` for range schedules.

### What issues does this PR fix or reference?

None

### Previous Behavior

`_handle_range` returned `None` in certain situations thus overwriting the `data` dictionary with `None`. Specifically, my range was hitting the else block on line 1222 and returning `None`.

### New Behavior

Never return anything and update the dictionary in place like it was doing before anyways. This makes the code less prone to errors if you forget to do a `return`.

### Tests written?

No

### Commits signed with GPG?

Maybe

### CODE REVIEW

File: salt/utils/schedule.py [hunk 1]
```diff
@@ -1203,15 +1202,15 @@ def _handle_range(job, data):
                             data['_error'] = ('Invalid date string for start. '
                                               'Ignoring job {0}.'.format(job))
                             log.error(data['_error'])
-                            return data
+                            return None
```
Comment (s0undt3ch) [inline salt/utils/schedule.py]: Just `return` no need to `return None`. Linters will likely complain.

Comment (garethgreenaway) [reply salt/utils/schedule.py]: Or just remove the returns.

Comment (pizzapanther) [reply salt/utils/schedule.py]: Returns are needed to exit early, I'll clean up the nones

**Comment by s0undt3ch** [comment]
These commits are now included in https://github.com/saltstack/salt/pull/46787. Closing.

**Pull request closed without merging**